main.bib

```diff
@@ -804,7 +804,8 @@ @book{lorenz1994object
 }
 
 @inproceedings{brito1994object,
-  author = {Brito, Fernando and dos Santos Carapu{\c{c}}a, Rog{\'e}rio},
+  author = {Brito, Fernando and dos Santos Carapua, Rogerio},
```

Comment:
@knowtempl this is wrong, see: https://d1wqtxts1xzle7.cloudfront.net/50115073/4ICSQ-libre.pdf?1478322418=&response-content-disposition=inline%3B+filename%3DObject_Oriented_Software_Engineering_Mea.pdf&Expires=1712925915&Signature=LRxVE1vvsfGOHMIFg-Y-ApTwiYHh1Rg1PdgYtFDAoYTtKulNN52InhEzY10jEfAr25cdEH6jf9Y5qShsvM4olMTUMtx5rrB-HqFjgKy8oUAnwyQmEpqN0Z5K3p~gVEH3is-iIzAunENltGBF2-JGGbM4Pchlhr2uPAoUc93ApAriMFbldZ4YeGfaGaz1bIP61S4vrMTeLO8qNpTHcRZpVYoI5qVokkZ~UQp8BoME0mHVVSfzFctKsairSXYPpF7RTvJq6FGc~Nyzz3Wfg6dHCKdu4Teai07ULEDYLi6l2y4MuBEp10oZI1tpW84KnvSZWaVRMY2TH26GCpp06J38HQ__&Key-Pair-Id=APKAJLOHF5GGSLRBV4ZA